demo/index.html

```diff
@@ -167,7 +167,7 @@
 						<template is="dom-bind" id="attrForhashParam">
 							<paper-input value="{{ hash }}" label="Hash"></paper-input>
 							<cosmoz-tabs _route-hash="{{ hash }}" hash-param="selected-tab" attr-for-hash-param="tab-hash" selected="1">
-								<cosmoz-tab heading="Tab1" tab-hash="first-tab">
+								<cosmoz-tab heading="Tab1" tab-hash="second-tab">
```

Comment:
Is it on purpose that Tab1 and Tab2 have the same `tab-hash` attribute ?
With that config, one cannot select Tab2.

Reply:
It is a mistake fixed that, rebased, updated demo and analysis.json.